sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -387,8 +412,39 @@ public synchronized void onActivityCreated(
     }
   }
 
+  @Override
+  public void onActivityPostCreated(
+      final @NotNull Activity activity, final @Nullable Bundle savedInstanceState) {
+    if (appStartSpan == null) {
+      activityLifecycleMap.remove(activity);
+      return;
+    }
+
+    final @Nullable ActivityLifecycleTimeSpan timeSpan = activityLifecycleMap.get(activity);
+    if (timeSpan != null) {
+      timeSpan.getOnCreate().stop();
+      timeSpan.getOnCreate().setDescription(activity.getClass().getName() + ".onCreate");
+    }
+  }
+
+  @Override
+  public void onActivityPreStarted(final @NotNull Activity activity) {
+    final long now = SystemClock.uptimeMillis();
+    if (appStartSpan == null) {
+      return;
+    }
+    final @Nullable ActivityLifecycleTimeSpan timeSpan = activityLifecycleMap.get(activity);
+    if (timeSpan != null) {
+      timeSpan.getOnStart().setStartedAt(now);
+    }
```

Comment:
`nit`: `SystemClock.uptimeMillis();` could be moved inside the if-check
```suggestion
    if (appStartSpan == null) {
      return;
    }
    final @Nullable ActivityLifecycleTimeSpan timeSpan = activityLifecycleMap.get(activity);
    if (timeSpan != null) {
      timeSpan.getOnStart().setStartedAt(SystemClock.uptimeMillis());
    }
```